Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -755,6 +784,15 @@ public extension Purchases {
     @objc(checkTrialOrIntroDiscountEligibility:completion:)
     func checkTrialOrIntroDiscountEligibility(productIdentifiers: [String],
                                               completion: @escaping ([String: IntroEligibility]) -> Void) {
+        guard !self.systemInfo.dangerousSettings.minimalImplementationOnly else {
+            var result: [String: IntroEligibility] = [:]
+            for product in productIdentifiers {
+                result[product] = IntroEligibility(eligibilityStatus: .unknown)
+            }
+            completion(result)
+            return
```

Comment:
I still need to add logging to these, but specifically _only_ to things that are called explicitly by the user, and not stuff that would otherwise have been called automatically.